specs/UrbanMobility/ArrivalEstimation/doc/spec.md

```diff
@@ -0,0 +1,67 @@
+# ArrivalEstimation
+
+## Description
+
+This Entity Type captures the estimation of arrival of a public transport vehicle, performing service on a route, to a certain stop. 
+
+## Data Model
+
++ `id`: Entity Id
+  + It shall be `urn:ngsi-ld:gtfs:ArrivalEstimation:<identifier>`. 
+
++ `type`: Entity Type 
+  + It shall be equal to `ArrivalEstimation`
+  
++ `dateCreated` : Entity's creation timestamp.
+  + Attribute type: [DateTime](https://schema.org/DateTime)
+  + Read-Only. Automatically generated. 
+ 
++ `dateModified` : Last update timestamp of this Entity.
+  + Attribute type: [DateTime](https://schema.org/DateTime)
+  + Read-Only. Automatically generated.
+  
++ `hasStop` : Stop to which this estimation applies to.
+  + Attribute type: Relationship. It shall point to an Entity of Type [gtfs:Stop](../../doc/Stop/spec.md)
+  + Mandatory
+  
++ `hasTrip` : The trip to which this estimation applies to. 
+  + Attribute type: Relationship. It shall point to an Entity of Type [gtfs:Trip](../../doc/Trip/spec.md)
+  + Mandatory
+
++ `remainingTime`: It shall contain the remaining time of arrival for the trip heading to the concerned stop. 
+  + Attribute type: Property. [Text](https://schema.org/Text). Remaining time shall be encoded as a ISO8601 duration. Ex. ."PT8M5S"`. 
+  + Attribute Metadata:
+    + `timestamp` (mapped to `observedAt` in NGSI-LD). Timestamp of the last attribute update
+      + Type: `[DateTime](https://schema.org/DateTime)
+      + Mandatory
+  + Mandatory
+  
++ `remainingDistance`: It shall contain  the remaining distance (in meters) of arrival for the trip heading to the concerned stop. 
+  + Attribute type: Property. Positive Number. [https://schema.org/Number](https://schema.org/Number)
+  + Attribute metadata:
```

Comment:
Should the unit of distance be added to the meta data? It is only "by convention" that the distance is meters.
Maybe it could be added as  an optional metadata field.

Reply:
as per the guidelines: 

Define a default unit for magnitudes. Normally it will be the unit as stated by the international system of units.

If a quantity is expressed in a different unit than the default one, use the unitCode metadata attribute.

https://github.com/Fiware/dataModels/blob/master/specs/guidelines.md

Reply:
Thanks for the clarification - NTC